config/schema.yaml

```diff
@@ -281,6 +283,299 @@ CLEAN_STATIC_FILES:
+  stages:
+    - deploy
+  default:
+    deploy: []
+  examples:
+    - stage:
+        deploy:
+          SEARCH_CONFIGURATION:
+            _merge: false if true configuration merging with default
+            engine: elasticsearch
+            elasticsearch_server_hostname: hostname
+            elasticsearch_server_port: '123456'
+            elasticsearch_index_prefix: magento
+            elasticsearch_server_timeout: '15'
+ELASTICSUITE_CONFIGURATION:
+  description: "Customize ElasticSuite service configuration stored in the 'system/default/smile_elasticsuite_core_base_settings'
+    section of the config.php file. To replace the existing configuration, specify values for each configuration option
+    required for your environment. To modify the existing configuration, specify values only for the options to add or update.
+    Then, add the `_merge: true` option."
+  magento_version: '>=2.2.0'
+  type: array
+  stages:
+    - deploy
+  default:
+    deploy: []
+  examples:
+    - comment: Default value depends on template configuration of ElasticSearch (i.e. default number of replicas and shards)
+        and ElasticSearch host.
+      stage:
+        deploy:
+          ELASTICSUITE_CONFIGURATION:
+            _merge: false if true configuration merging with default"
+            es_client:
+              servers: 'remote-host:9200'
+            indices_settings:
+              number_of_shards: 3
+              number_of_replicas: 3
+QUEUE_CONFIGURATION:
+  description: "Replace or modify the AMQP configuration generated during the deployment process. This configuration
+    is generated only if your project has been configured with a RabbitMQ service. To replace the existing
+    configuration, specify values for each configuration option required for your environment.
+    To modify the existing configuration, specify values only for the options to add or update.
+    Then, add the `_merge: true` option."
+  type: array
+  stages:
+    - deploy
+  default:
+    deploy: []
+  examples:
+    - stage:
+        deploy:
+          QUEUE_CONFIGURATION:
+            _merge: false if true configuration merging with default
+            amqp:
+              host: test.host
+              port: 1234
+            amqp2:
+              host: test.host2
+              port: 12345
+            mq:
+              host: mq.host
+              port: 1234
+CACHE_CONFIGURATION:
+  description: "Replace or modify the Magento cache configuration generated during the deployment process.
+    To replace the existing configuration, specify values for each configuration option required for your environment.
+    To modify the existing configuration, specify values only for the options to add or update. Then, add the `_merge: true` option."
+  type: array
+  stages:
+    - deploy
+  default:
+    deploy: []
+  examples:
+    - stage:
+        deploy:
+          CACHE_CONFIGURATION:
+            _merge: false if true configuration merging with default
```

Comment:
Same here, these options should be boolean

Reply:
Moved `if true configuration merging with default` to the example comment section